src/test/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtilsUnitTest.java

```diff
@@ -808,6 +808,24 @@ public void testFindNumberOfRepetitions() throws Exception {
         Assert.assertEquals(GATKVariantContextUtils.findNumberOfRepetitions("AT".getBytes(), "".getBytes(), false), 0); //empty test string
     }
 
+    @Test
+    public void testFindNumberOfRepetitionsFullArray() throws Exception {
+        Assert.assertEquals(GATKVariantContextUtils.findNumberOfRepetitions("XXXATG".getBytes(),3 ,3, "ATGATGATGATGYYY".getBytes(), 0, 12, true),4);
```

Comment:
\[FindBugs\] WARNING: Dm: Found reliance on default encoding in org\.broadinstitute\.hellbender\.utils\.variant\.GATKVariantContextUtilsUnitTest\.testFindNumberOfRepetitionsFullArray\(\): String\.getBytes\(\)